tpm2/include/kmyth.h

```diff
@@ -22,30 +22,36 @@
  * @param[in]  output_path       Path to .ski file where the kmyth-seal output
  *                               will be written
  *
- * @param[in]  auth_string       Authorization string to be applied to the
+ * @param[in]  auth_bytes        Authorization string to be applied to the
```

Comment:
I will just make this comment/question once, but it applies almost universally to the header file comments for auth_bytes. In our mains, we accept an 'authString' (char *) command line input from the user, but, everywhere else, I believe that this value is treated as a more generic byte array. I'm thinking, therefore, that in the header file comments describing this parameter, we should not be using the 'authorization string' description, but rather wording to convey the potential use of a more generic 'key' or 'secret' like 'authorization bytes" or "authorization secret", or "authorization parameter" or "authorization byte array" or ???. The TPM uses the term "authorization value", so I would not overload that. If you think that "string" is a sufficiently generic description, I could probably be convinced of that as well, though.